examples/record-type-reference/record_type_reference.bal

```diff
@@ -0,0 +1,42 @@
+import ballerina/io;
+
+type Person record {
+    string name;
+    int age = 25;
+    !...
+};
+
+type Employee record {
+    // Referencing the `Person` record
```

Comment:
Should this be a comment for `Employee` instead?

```ballerina
// Define a record `Employee` referencing the `Person` record.
type Employee record {
```

Reply:
Hmm not really right? What type referencing does is copy the fields of the specified fields to the new record. And there can be multiple such references.

Reply:
I think it is better to add some kind of comment at the top as well.